### DISCUSSION TITLE
apache folder location nextcloud alpine lxc

### DISCUSSION THREAD

**Discussion by Citroene** [tteck/Proxmox#2048] [Help Needed]
Hi,

I can't find the location of the apache dir.
the defauld location is /etc/apache2 but there is no apache dir in etc.

**Comment by Vakrehus** [comment]
You're right.
There is a nginx folder.

/etc/nginx

**Comment by vahidirp** [comment]
In a standard installation of Nextcloud on Alpine Linux within an LXC (Linux Container), the Apache (or Nginx) configuration files and web server-related files are typically located in the following directories:

    Apache Configuration:
        The Apache configuration files are usually stored in /etc/apache2/ or /etc/httpd/ on Alpine Linux, depending on the version and specific configuration.

    Nextcloud Installation:
        The Nextcloud files are typically located in the web server's document root directory. This is often in a path like /var/www/ or /usr/share/, and within it, you would find a directory named after your Nextcloud installation.

    Data Directory:
        The data directory, where Nextcloud stores user data and files, is often configured during the installation process. By default, it may be something like /var/www/nextcloud/data/.

It's important to note that the specific paths may vary depending on your installation method and configuration choices. Here's a generic example of where these files might be located:

    Apache configuration files: /etc/apache2/ or /etc/httpd/
    Nextcloud installation: /var/www/nextcloud/
    Nextcloud data directory: /var/www/nextcloud/data/

To find the exact paths on your system, you can check the Apache configuration file (/etc/apache2/apache2.conf or /etc/httpd/httpd.conf) for the DocumentRoot directive. This directive specifies the root directory of the web server.

check these : 
```
# Check Apache configuration file for DocumentRoot
cat /etc/apache2/apache2.conf | grep DocumentRoot

# Navigate to the web server root directory
cd /var/www/nextcloud/

# List the contents of the Nextcloud directory
ls /var/www/nextcloud/

# Check the data directory location
cat /var/www/nextcloud/config/config.php | grep datadirectory
```
